bundler/spec/other/major_deprecation_spec.rb

```diff
@@ -587,8 +583,8 @@
       end
 
       it "prints a deprecation warning informing about its removal", :bundler => "< 3" do
-        expect(deprecations).to include("the `--outdated` flag to `bundle show` was undocumented and will be removed without replacement")
-      end
+       expect(deprecations).to include("The `--outdated` flag to `bundle show` was undocumented and will be removed without replacement")
```

Comment:
Fixing the case seems fine, but this is also removing one whitespace from the beginning of the line, making the indentation inconsistent. Can you fix it?